modules/st2-auto-form/modules/st2-form-text-field/template.html

```diff
@@ -0,0 +1,29 @@
+<label class="st2-auto-form__label st2-auto-form__text-field"
+  ng-class="{'st2-auto-form--required': spec.required}">
+
+  <div class="st2-auto-form__title"
+      ng-if="spec.name || name">
+    {{ spec.name || name }}
+  </div>
+  
+  <textarea
+    class="st2-auto-form__field st2-elastic-field"
+    placeholder="{{spec.default}}"
+    ng-required="spec.required"
+    ng-model="rawResult"
+    ng-model-options="options || {}"
+    ng-pattern="spec.pattern"
+    ng-disabled="disabled"
+    minlength="{{spec.minLength}}"
+    maxlength="{{spec.maxLength}}"
+    btsx-value-format="{{spec.format}}"
+    st2-elastic-field-min-rows="{{spec.minRows}}"
+    st2-elastic-field-max-rows="{{spec.maxRows}}">
```

Comment:
For one, I don't think that min/maxRows should be a part of the spec. Spec mostly defines the data and its constraints, not representation. Instead, we should probably make this parameters scope attributes of st2FormTextField and hardcode them for now at this level.